input_example.txt

```diff
@@ -2,3 +2,5 @@
 disk = '/home'
 wsize = '1m'
 period = '5'
+io_type.cache_io.mode='-C'
+*.fs_type.*.exclud='1-500'
```

Comment:
`io_type.cache_io.mode = '-C'`
` *.fs_type.*.exclud = '1-500' `
for better readability and uniformity